components/sponsor/NftSubmissionsList.vue

```diff
@@ -0,0 +1,213 @@
+<script setup lang="ts">
+import type { NftSubmission } from '~/types';
+import { formatDate } from '~/utils/date';
+import { getTierMedal } from '~/utils/nft-tiers';
+
+interface Props {
+  address: string | undefined;
+  isConnected: boolean;
+}
+
+const props = defineProps<Props>();
+
+const emit = defineEmits<{
+  'edit-submission': [submission: NftSubmission];
+  'close': [];
+}>();
+
+const { t } = useI18n();
+const { submissions, isLoading: isLoadingSubmissions, error: submissionsError, fetchSubmissions } = useNftSubmissions();
+const { open: openWalletConnect } = useWeb3Connection();
+const { fetchNftMetadata } = useNftMetadata();
+
+const nftMetadataCache = ref<Record<number, { tier?: string; releaseName?: string }>>({});
+
+async function loadSubmissions(): Promise<void> {
+  if (!props.isConnected || !props.address) {
+    return;
+  }
+  await fetchSubmissions(props.address);
+}
+
+async function connectWallet(): Promise<void> {
+  try {
+    await openWalletConnect();
+  }
+  catch (error) {
+    console.error('Failed to connect wallet:', error);
```

Comment:
same here in general any console.error introduced in the PR it would make sense to replace them with logger.error